kalite/updates/download_track.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from django.conf import settings
+logging = settings.LOG
+
+from .settings import VIDEO_DOWNLOAD_QUEUE_FILE
+
+class VideoQueue:
```

Comment:
Class to manage video downloads. It works, but implementation comments/feedback appreciated here!